File: C#.Net/Tests/JDI Tests/Tests/Epam/FormExamples.cs
using JDI_Tests.Entities;
using JDI_Tests.Epam_UIObjects;
using JDI_Tests.Epam_UIObjects.DataProviders;
using NUnit.Framework;

namespace JDI_Tests.Tests.Epam
{
    [TestFixture]
    public class FormExamples
    {
        [SetUp]
        public void SetUp()
        {
            EpamSite.JobDescriptionPage.IsOpened();
        }

        [Test]
        public void FillFormExample([ValueSource(typeof(AttendeesProvider), "Attendees")] Attendee attendee)
        {
            EpamSite.JobDescriptionPage.AddCvForm.Submit(attendee);
            // It's not stable, GetAttribute() sometimes returnes all classes and sometimes only a first part 'required '
            Assert.That(EpamSite.JobDescriptionPage.Captcha.GetAttribute("class"), Does.Contain("form-field-error"));
        }
    }
}

File: Java/Tests/jdi-uitest-tutorialtests/src/test/java/com/epam/jdi/uitests/testing/simple/examples/FormExamples.java
package com.epam.jdi.uitests.testing.simple.examples;

import com.epam.jdi.dataProviders.AttendeesProvider;
import com.epam.jdi.entities.Attendee;
import com.epam.jdi.uitests.testing.TestsBase;
import com.epam.web.matcher.testng.Assert;
import org.testng.annotations.BeforeMethod;
import org.testng.annotations.Test;

import java.io.IOException;
import java.lang.reflect.Method;

import static com.epam.jdi.site.epam.EpamSite.jobDescriptionPage;


public class FormExamples extends TestsBase {

    @BeforeMethod
    public void before(Method method) throws IOException {
        jobDescriptionPage.shouldBeOpened();
    }
    @Test(dataProvider = "attendees", dataProviderClass = AttendeesProvider.class)
    public void fillForm(Attendee attendee) {
        jobDescriptionPage.addCVForm.submit(attendee);
        Assert.contains(() -> jobDescriptionPage.captcha.getAttribute("class"), "form-field-error");
    }
}
